### PR TITLE
feat(aws): Add new package @tsed/platform-aws

### PR DESCRIPTION
**Pull request by Romakita**
## Informations

Type | Breaking change
---|---
Feature | No

****

## Description
A package to configure Ts.ED with AWS

## Usage

Create a new `LambdaServer.ts` in `src` directory:

```typescript
import {PlatformAws} from "@tsed/platform-aws";
import "@tsed/platform-express" 
// or import "@tsed/platform-koa"

PlatformAws.bootstrap(Server, {
 aws: {
   binaryMimeTypes: [ // optional
     // mime types list 
   ]
 },
 // additional Ts.ED options. See https://tsed.io/tutorials/aws.html
});

// Handler used by AWS
export const handler = PlatformAws.callback();
```

Then create `lambda.js` on your root project:

```javascript
module.exports = require("./dist/LambdaServer.js");
```

This file will be used by AWS to forward request to your application.

Finally, [package and create your Lambda function](http://docs.aws.amazon.com/lambda/latest/dg/nodejs-create-deployment-pkg.html), 
then configure a simple proxy API using Amazon API Gateway and integrate it with your Lambda function.

See more details on [`aws-serveless-express`](https://github.com/awslabs/aws-serverless-express) project.

## Getting the API Gateway event object

This package includes decorators to easily get the event object Lambda receives from API Gateway:

```typescript
import {Controller, Get} from "@tsed/common"; 
import {AwsEvent, AwsContext} from "@tsed/platform-aws"; 

@Controller('/')
class MyCtrl {
 @Get('/')
 get(@AwsEvent() event: any, @AwsContext() context: any) {
   console.log("Event", apiGateway.event);
   console.log("Context", apiGateway.context);
   
   return apiGateway;
 }
}
```

## Todos

- [x] Tests
- [x] Coverage
- [x] Example
- [x] Documentation

### CODE REVIEW

File: packages/platform-aws/test/app/Server.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  mount: {
+    "/": [AwsCtrl]
+  }
+})
+export class Server {
+  @Inject()
+  app: PlatformApplication;
+
+  /**
+   * This method let you configure the middleware required by your application to works.
```
Comment (LorianeE) [inline packages/platform-aws/test/app/Server.ts:31]: This method let**s** you configure the middleware required by your application to **work**.

**Review by LorianeE — CHANGES_REQUESTED** [review]

File: packages/platform-aws/test/app/Server.ts (hunk 1 above)
Comment (Romakita) [reply packages/platform-aws/test/app/Server.ts]: done

File: packages/platform-aws/readme.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+This file will be used by AWS to forward request to your application.
+
+Finally, [package and create your Lambda function](http://docs.aws.amazon.com/lambda/latest/dg/nodejs-create-deployment-pkg.html), 
+then configure a simple proxy API using Amazon API Gateway and integrate it with your Lambda function.
+
+See more details on [`aws-serveless-express`](https://github.com/awslabs/aws-serverless-express) project.
+
+## Getting the API Gateway event object
+
+This package includes decorators to easily get the event object Lambda receives from API Gateway:
```
[the remark this answers is not in the document]
Comment (Romakita) [reply packages/platform-aws/readme.md]: Done

File: packages/platform-aws/test/app/Server.ts (hunk 1 above)
Comment (LorianeE) [reply packages/platform-aws/test/app/Server.ts]: Bah tu l'as carrément enlevé ?

**Pull request merged**